src/model/RestAction.ts

```diff
@@ -220,6 +230,24 @@ class RestAction implements Action {
                   );
         };
 
+        const getClientCertificationConfiguration = (
+            clientCertificatePath,
+            clientKeyPath,
+        ) => {
+            try {
+                const cert = readFileSync(clientCertificatePath);
+                const key = readFileSync(clientKeyPath);
+                return {
+                    cert,
+                    key,
+                    rejectUnauthorized: false,
+                };
+            } catch (error) {
```

Comment:
What kind of errors can happen here? With the current implementation, if an error happens here, doesn't this mean the action cannot be executed as was expected by the user (client cert is not used) and the action should fail? (Especially because there also is an error log which usually only happens if an action / scenario fails)